### PR TITLE
Release v1.3.50.3

### PR DESCRIPTION
**Pull request by iulianpascalau**
- migration work to multiverx organization

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,16 @@
...
+  src="https://raw.githubusercontent.com/multiversx/mx-chain-go/master/multiversx-logo.svg"
+  alt="MultiversX">
 </div>
 <br>
 
 <br>
 
-[![](https://img.shields.io/badge/made%20by-Elrond%20Network-blue.svg?style=flat-square)](http://elrond.com/)
-[![](https://img.shields.io/badge/project-Elrond%20Network%20Mainnet-blue.svg?style=flat-square)](http://elrond.com/)
+[![](https://img.shields.io/badge/made%20by-MultiverX-blue.svg?style=flat-square)](http://multiversx.com/)
```
Comment (mariusmihaic) [inline README.md:10]:
```suggestion
[![](https://img.shields.io/badge/made%20by-MultivesrX-blue.svg?style=flat-square)](http://multiversx.com/)
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,16 @@
 <div style="text-align:center">
   <img
-  src="https://raw.githubusercontent.com/ElrondNetwork/elrond-go/master/elrond_logo_01.svg"
-  alt="Elrond Network">
+  src="https://raw.githubusercontent.com/multiversx/mx-chain-go/master/multiversx-logo.svg"
```
Comment (mariusmihaic) [inline README.md:3]: [question]

Was this logo svg file also renamed?

**Review by andreibancioiu — DISMISSED** [review]

**Review by sasurobert — COMMENTED** [review]
Only what Marius commented

File: README.md (hunk 2 above)
Comment (iulianpascalau) [reply README.md:3]: https://github.com/multiversx/mx-chain-go/blob/master/multiversx-logo.svg

File: README.md (hunk 1 above)
Comment (iulianpascalau) [reply README.md:10]: fixed

**Review by mariusmihaic — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**